cli/src/index.ts

```diff
@@ -582,14 +588,15 @@ yargs(hideBin(process.argv))
             const network = deployments.network as Network;
             const deps: Partial<{ [C in Chain]: Deployment<Chain> }> = await pullDeployments(deployments, network, verbose);
             const signerType = argv["signer-type"] as SignerType;
+            const payerPath = argv["payer"] === false ? undefined : argv["payer"];
```

Comment:
this is redundant if we remove the `default: false` above we can then just have
```suggestion
            const payerPath = argv["payer"];
```
instead (or even inline the use of `payerPath`)